manifests/controller/cdi-controller-deployment.yaml

```diff
@@ -1,3 +1,36 @@
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: cdi-sa
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: cdi
+rules:
+- apiGroups: [""]
+  resources: ["persistentvolumeclaims"]
+  verbs: ["get", "list", "watch", "create", "update", "patch"]
+- apiGroups: [""]
+  resources: ["pods"]
+  verbs: ["get", "list", "watch", "create"]
+- apiGroups: [""]
+  resources: ["namespaces"]
```

Comment:
why do we need namespaces? Is it so that we can watch pvc and pods in many namespaces?

Reply:
I don't know, was thinking there might be a time when we need to generate something in a new namespace, we probably really don't need it for current functionality

Reply:
if you want me to remove, I can, it's your call